stubs/app/Http/Requests/CreateResourceCollectionRequest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Http\Requests;
+
+use CodeZero\UniqueTranslation\UniqueTranslationRule;
+use Illuminate\Foundation\Http\FormRequest;
+
+class CreateResourceCollectionRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return $this->user()->id === $this->user_id;
```

Comment:
This should return true because the resource collection does not yet exist; in the `routes/resource-collections.php` middleware, you are already checking to make sure that the current user can create a Resource Collection and that is sufficient.